packages/core/src/tracing/instrumentation/messaging/rdkafka.js

```diff
@@ -44,17 +44,23 @@ exports.activate = function activate(extraConfig) {
     }
     extraConfigHeader = extraConfig.tracing.kafka.headerFormat;
   }
-  logWarningForKafkaHeaderFormat(extraConfigHeader || configHeader);
+  if (extraConfigHeader || configHeader) {
```

Comment:
q: Why this change?

Reply:
The function should only be called if a valid Kafka header is present.

Reply:
`configHeader` should have always a value because we set a default value in the normalize fn.
Same for https://github.com/instana/nodejs/pull/1311/files#diff-d11898bc026195d9d4ad65d0a381844c1bc6daf3187a71ad6b7feaf62123dd7eR46.